specification/ml/_types/Analysis.ts

```diff
@@ -50,7 +50,7 @@ export class AnalysisConfig {
   /**
    * A comma separated list of influencer field names. Typically these can be the by, over, or partition fields that are used in the detector configuration. You might also want to use a field name that is not specifically named in a detector, but is available as part of the input data. When you use multiple detectors, the use of influencers is recommended as it aggregates results for each influencer entity.
    */
-  influencers: Field[]
+  influencers?: Field[]
```

Comment:
This will always be present when getting the job definitions from the get jobs endpoint.  It is not compulsory when creating a job.